R/functions_effects_DyNAM_choice.R

```diff
@@ -167,7 +167,7 @@ init_DyNAM_choice.default <- function(
 #' @noRd
 #'
 #' @examples
-#' \dontrun{
+#' \donttest{
```

Comment:
\donttest is still run as examples, perhaps takes too long for CRAN?

Reply:
I'll be worried about the examples in `estimate()`. In the effects files, the examples expend less than a second. Did you find one that takes too long to run here?

Reply:
Let's try it then.